app/features/build_runner/build_runner.rb

```diff
@@ -214,7 +214,7 @@ def prepare_build_object
         project: self.project,
         number: new_build_number,
         status: :pending,
-        timestamp: Time.now,
+        timestamp: Time.now.utc,
```

Comment:
Could you add a comment to the code here on why this change is necessary? I assume this depends on the server's time zone?

Reply:
exactly.